src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -864,7 +864,11 @@ public function relation(Request $request)
                     }
                 } else {
                     $total_count = $model->count();
-                    $relationshipOptions = $model->take($on_page)->skip($skip)->get();
+                    if (isset($options->whereRaw)) {
+                        $relationshipOptions = $model->whereRaw($options->whereRaw)->take($on_page)->skip($skip)->get();
+                    } else {
+                        $relationshipOptions = $model->take($on_page)->skip($skip)->get();
+                    }
```

Comment:
I think this can be simplified like this. We also need documentation for this.

```suggestion
                    if (isset($options->whereRaw)) {
                        $model = $model->whereRaw($options->whereRaw)->take($on_page)->skip($skip)->get();
                    }
                    $relationshipOptions = $model->take($on_page)->skip($skip)->get();
```